src/lib.rs

```diff
@@ -188,9 +191,19 @@ impl<T: Send> ThreadLocal<T> {
     }
 
     /// Returns the element for the current thread, if it exists.
+    #[cfg(feature = "nightly")]
+    pub fn get(&self) -> Option<&T> {
+        match unsafe { thread_id::THREAD_HOLDER.as_ref() } {
+            None => None,
+            Some(x) => self.get_inner(&x.0),
+        }
+    }
```

Comment:
Can this logic be moved into a safe wrapper in the `thread_id` module?

Reply:
I don't think that the logic can easily be moved (because it is now closely linked with the ThreadLocal struct itself which even modifies the THREAD_HOLDER internally) but i can add a safe `getter` and an unsafe `setter`, so the unsafety stays contained inside `thread_id` (and the single set call we have inside `insert`) this way we could even remove the ThreadHolder import, although thread_id will still leak it through the get call